Python/Product/Debugger/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -1481,5 +1481,9 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
             }
             return null;
         }
+
+        internal async void RefreshThreadFrames(long threadId) {
+            await _process.GetThreadFramesAsync(threadId);
+        }
```

Comment:
Because this is async void it's going to be a fire and forget call...  So while we await on the threads coming back here, we don't await on them coming back DTEDebuggerExtensions.  Seems like we need this to return a Task and we need to do a .Wait() over in DTEDebuggerExtensions, and we might need to do a .ConfigureAwait(false) here to make sure we don't deadlock